containers/hpu/Containerfile

```diff
@@ -25,17 +25,16 @@ COPY containers/bin/debug-* ${VIRTUAL_ENV}/bin/
 COPY requirements.txt requirements-hpu.txt /tmp
 RUN sed 's/\[.*\]//' /tmp/requirements.txt >/tmp/constraints.txt && \
     export PIP_NO_CACHE_DIR=off; \
-    ${VIRTUAL_ENV}/bin/pip install wheel && \
-    CMAKE_ARGS="" \
-        CFLAGS="-mno-avx" \
+    ${VIRTUAL_ENV}/bin/pip install -U wheel pip && \
+    CMAKE_ARGS="-DLLAMA_NATIVE=off" \
         FORCE_CMAKE=1 \
         ${VIRTUAL_ENV}/bin/pip install --no-binary llama_cpp_python -c /tmp/constraints.txt llama_cpp_python && \
     ${VIRTUAL_ENV}/bin/pip install -r /tmp/requirements.txt -r /tmp/requirements-hpu.txt && \
     rm /tmp/constraints.txt && \
     find ${VIRTUAL_ENV} -name __pycache__ | xargs rm -rf
 
 COPY . /tmp/instructlab
-RUN ${VIRTUAL_ENV}/bin/pip install "/tmp/instructlab[habana]" && \
+RUN ${VIRTUAL_ENV}/bin/pip install "/tmp/instructlab[hpu]" && \
```

Comment:
Is habana being renamed hpu? I'm not too familiar with all the acronyms so I'm wondering why do we have to change this?

Reply:
`hpu` is PyTorch's name for the device. Intel prefers that we no longer use the internal names `habana` or `Habana Labs` to refer to Intel Gaudi systems.

Reply:
I'll clean up the documentation in another PR when I finalize the move to 1.16.0.

Reply:
Sounds good!